### PR TITLE
Update defining-a-component.md

### PR DESCRIPTION
**Pull request by timrosenblatt**
I was looking at the guide and realized this could be written in a more straightforward way. I added a bit of clarity with respect to path names vs file names

### CODE REVIEW

File: source/components/defining-a-component.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-To define a component, create a template whose name starts with
-`components/`. To define a new component, `{{blog-post}}` for example,
+To define a component, create a template in the `components/` directory.
```
Comment (michaelrkn) [inline source/components/defining-a-component.md]: I think it would be even more clear to say "in the `app/templates/components/` directory".

**Comment by michaelrkn** [comment]
Thanks for the suggestion! You're right, this isn't very clear currently.

**Comment by locks** [comment]
The whole start of this guide is confusing, I'll try reviewing it shortly

**Comment by michaelrkn** [comment]
@locks https://github.com/emberjs/guides/pull/784#discussion_r40170578

**Pull request merged**